### ISSUE TITLE
 [a11y]: Progressbar [example] - loading message bug

### ISSUE THREAD

**Issue by andreancardona** [carbon-design-system/carbon#15263]
### Package

@carbon/react

### Browser

FF, Safari & iPhone

### Package version

@carbon/react@1.41.0

### Automated testing tool and ruleset

Manual QA

### Assistive technology

VO & JAWS

### Package

@carbon/react

### Browser

FF, Safari & iPhone

### Package version

@carbon/react@1.40.0

### Automated testing tool and ruleset

Manual QA

### Assistive technology

VO & JAWS

### Description
Exporting Data Loading status not conveyed by Screen Reader

Once we hit on Example Link and press enter OR space bar Exporting Data Loading Message starts but Screen Reader doesn't announce the loading status for exporting / Complete. Visually it shows 20MB, 50MB etc till 720MB.

### Recommended Fix
Screen Reader should instantly announce the loading status for the user.

If the screen reader does not announce the status change in a page element, it’s difficult for users to understand it on their own. This defect mainly impacts users who are blind. 

### WCAG 2.1 Violation
4.1.3 Status Messages

**Comment by 2nikhiltom** [comment]
The issue was retested  (in @carbon/react v1.84), and the issue has been fixed.
